web/src/e2e/Route.e2e.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable import/no-extraneous-dependencies */
+const puppeteer = require('puppeteer');
+
+const {
+  setupLogin
+} = require('./service')
+
+let browser;
+const domSelectors = {
+  inputRoutename: '#name',
+  inputRoutepath: '#uris_0',
+  inputNodehost: '#nodes_0_host',
+  inputNodeport: '#nodes_0_port',
+  inputNodeweight: '#nodes_0_weight',
+  routeList: '.ant-layout-sider-children ul li:nth-child(2)',
+  buttonCreateroute: '.ant-pro-table-toolbar-option button',
+  buttonNext: '.ant-row-end .ant-col:nth-child(2)',
+  successIco: '.ant-result-success .anticon-check-circle',
+  buttonReturnRoutelist: '.ant-result-extra .ant-btn-primary'
+};
+
```

Comment:
For me these are too difficult to maintain

Reply:
we need to find a better way.

Reply:
please take a look at https://github.com/getgauge/taiko

Reply:
> please take a look at https://github.com/getgauge/taiko

ok, will try to find a better way. refer issue: https://github.com/apache/apisix-dashboard/issues/1036